images/cassandra/rootfs/usr/local/bin/include/cassandra-seed.h

```diff
@@ -0,0 +1,67 @@
+/* Created by "go tool cgo" - DO NOT EDIT. */
+
+/* package command-line-arguments */
+
+/* Start of preamble from import "C" comments.  */
+
+
+#line 3 "/home/greenday/go/src/k8s.io/examples/cassandra/go/main.go"
```

Comment:
This should probably be changed, I guess? :)

Reply:
`/* Created by "go tool cgo" - DO NOT EDIT. */`
Use as is :smile: